sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -430,20 +466,21 @@ async Task<bool> OnBreakpointHit (SessionId sessionId, JObject args, Cancellatio
 		async Task OnDefaultContext (SessionId sessionId, ExecutionContext context, CancellationToken token)
 		{
 			Log ("verbose", "Default context created, clearing state and sending events");
-
+			if (contexts.TryGetValue (sessionId.sessionId ?? "default", out var oldContext)) {
```

Comment:
nitpick: maybe add `var id_to_use = sessionId.sessionId ?? "default";`, and use that in the two places in this method. Just to avoid accidental changes to only one of them.